prompttools/harness/chat_history_harness.py

```diff
@@ -16,10 +16,6 @@ class ChatHistoryExperimentationHarness(ExperimentationHarness):
     Args:
         model_name (str): The name of the model.
         chat_histories (List[List[Dict[str, str]]]): A list of chat histories that will be fed into the model.
-        use_scribe (Optional[bool], optional): Whether to use ``HegelScribe`` for logging and analytics.
```

Comment:
I don't think these are your changes. Could you rebase on `main` so that the diff is accurate?

Reply:
Hopefully should be fixed now. I reverted earlier due to fork being way behind. Did not notice the effect- apologies.